static/js/components/LearnerInfoCard.js

```diff
@@ -18,50 +22,85 @@ export default class LearnerInfoCard extends React.Component {
     profile: Profile,
     toggleShowPersonalDialog: () => void,
     toggleShowAboutMeDialog: () => void,
+    openLearnerEmailComposer: () => void,
   };
 
-  email = (email: string): React$Element<*> => (
-    <span className="profile-email">{email}</span>
-  );
+  ownProfilePage: boolean;
+
+  isOwnProfilePage = (): boolean => {
+    if (R.isNil(this.ownProfilePage)) {
+      this.ownProfilePage = isProfileOfLoggedinUser(this.props.profile);
+    }
+    return this.ownProfilePage;
+  };
```

Comment:
The repeated calculations are necessary though. React components should always assume their props can change at any moment. This calculation is not time consuming, it's just comparing the username of the profile and for the `SETTINGS.user`